stacks/amp.stack.yml

```diff
@@ -31,6 +31,26 @@ services:
       placement:
         constraints: [node.role == manager]
 
+  amplifier-gateway:
+    image: appcelerator/amplifier-gateway:${TAG:-latest}
+    networks:
+      default:
+        aliases:
+          - amplifier-gateway
+    labels:
+      io.amp.role: "infrastructure"
+    environment:
+      - "SERVICE_PORTS=80"
+      - "VIRTUAL_HOST=https://gw.*"
+    deploy:
+      mode: global
+      labels:
+        io.amp.role: "infrastructure"
+      restart_policy:
+        condition: on-failure
+      placement:
+        constraints: [node.role == manager]
```

Comment:
I don't think we need this constraint since it doesn't need access to the docker socket on a swarm manager and I don't believe there's any kind of guarantee that it communicates with the amplifier instance co-located on the same node for some kind of performance benefit. I believe it's fine to place the gateway on workers and let them scale independently from amplifier instances running on managers.